udata/templates/search.html

```diff
@@ -134,7 +138,15 @@
 {% block main_content %}
 <div class="tab-content">
 
-    <div class="tab-pane row active" id="datasets">
+    {% if not datasets and not reuses and not organizations and not users and not territories %}
```

Comment:
It looks fine

![capture d ecran 2016-04-04 a 11 14 01](https://cloud.githubusercontent.com/assets/3556/14243539/5eb9c2e2-fa56-11e5-91c6-470250846eb1.png)

I wasn't sure about the wording though.